api/api.go

```diff
@@ -52,15 +52,15 @@ func New(
 
 	router := mux.NewRouter()
 
-	// to serve api doc and swagger-ui
+	// to serve api docs
 	router.PathPrefix("/doc").Handler(
 		http.StripPrefix("/doc/", http.FileServer(http.FS(doc.FS))),
 	)
 
-	// redirect swagger-ui
+	// redirect api-docs
 	router.Path("/").HandlerFunc(
 		func(w http.ResponseWriter, req *http.Request) {
-			http.Redirect(w, req, "doc/swagger-ui/", http.StatusTemporaryRedirect)
+			http.Redirect(w, req, "doc/api-docs/", http.StatusTemporaryRedirect)
```

Comment:
`/doc/api-doc` seems a little bit strange, how about `/doc/api` or `doc/stoplight` ? Or do you have other thoughts?

Reply:
How about `/doc/stoplight-ui` to keep it consistent with swagger?

Reply:
I pushed `/doc/stoplight-ui`. Let me know if you prefer something else